spopt/locate/p_center.py

```diff
@@ -280,19 +280,28 @@ def facility_client_array(self) -> None:
 
             self.fac2cli.append(array_cli)
 
-    def solve(self, solver: pulp.LpSolver):
+    def solve(self, solver: pulp.LpSolver, results: bool = True):
         """
         Solve the PCenter model
 
         Parameters
         ----------
         solver: pulp.LpSolver
             solver supported by pulp package
+        
+        results: bool
+            if True it will create metainfo - which facilities cover which demand and vice-versa, and the uncovered demand - about the model results
 
         Returns
         -------
         PCenter object
         """
         self.problem.solve(solver)
         self.check_status()
+
+        if results:
+            self.facility_client_array()
+            self.client_facility_array()
+            self.uncovered_clients()
```

Comment:
With $p$-center and $p$-median there will be no uncovered clients. Same goes for Set Cover. The only model with uncovered clients will be the Max Cover.

Reply:
Great job, @gegen07